library/general/disallowedtags/suite.yaml

```diff
@@ -11,6 +11,10 @@ tests:
     object: samples/container-image-must-not-have-latest-tag/example_allowed.yaml
     assertions:
     - violations: no
+  - name: exempt-allowed
+    object: samples/container-image-must-not-have-latest-tag/example_allowed_excluded.yaml
+    assertions:
+    - violations: no
   - name: example-disallowed
```

Comment:
better names for these cases would make the suite.yaml much more helpful.  Right now, I have no way to know what's being tested other than to go look at the objects and grok the template.  Names that say _what_ we're testing would be helpful.

Reply:
Renamed test names and objects.